resources/js/Components/Form/ItemCreateForm.tsx

```diff
@@ -147,18 +151,21 @@ const ItemCreateForm = ({
                     </StyledGrid>
                 </Grid>
             </StyledBox>
-            <StyledFooter container>
+            <Grid container>
                 <Grid container spacing={2}>
                     <Grid item md={type === 'Edit' ? 4 : 2} xs={4}>
                         {type === 'Create' ? (
-                            childButton
+                            cancelFormButton
                         ) : (
                             <Button onClick={() => onCloseDrawer()} fullWidth>
                                 Cancel
                             </Button>
                         )}
                     </Grid>
-                    <Grid item md={type === 'Edit' ? 4 : 6} />
+                    <Grid item md={type === 'Edit' ? 4 : 2} xs={4}>
```

Comment:
let's use spacing here instead

Reply:
I'm not sure I understand.

Reply:
you can look at other footer in `CharacterCreateForm` for example